packages/RichText-Core.package/RTEAddComponentMenu.class/instance/closeAllButtonDropdowns.st

```diff
@@ -2,4 +2,4 @@ initialization
 closeAllButtonDropdowns 
 
 	super closeAllButtonDropdowns.
-	self hideCategory: self textButtons 
\ No newline at end of file
+	self hideCategory: (self buttonGroups at: #TextButtonGroup) 
```

Comment:
Why not iterate over the dictionary here to really close all dropdowns?
`aDictionary valuesDo: [:value | self hideCategory: value]`

Reply:
Because the other dropdowns are already closed with the method from its super class